src/server/middleware.ts

```diff
@@ -3,17 +3,19 @@ import * as path from 'path';
 import {NextFunction, Request, RequestHandler, Response} from 'express-serve-static-core';
 
 import {getJsonFile} from '../utils/getJsonFile';
-import {getManifest, ManifestFile, ServiceDeclaration, RewriteContext} from '../utils/getManifest';
 import {getProjectConfig} from '../utils/getProjectConfig';
 import {getProviderUrl} from '../utils/getProviderUrl';
+import {ManifestFile, PlatformManifest, ServiceDeclaration} from '../utils/manifests';
+import {getManifest, RewriteContext, getPlatformManifest} from '../utils/getManifest';
 
 /**
  * Creates express-compatible middleware function that will add/replace any URL's found within app.json files according
  * to the command-line options of this utility.
  */
-export function createAppJsonMiddleware(providerVersion: string, runtimeVersion?: string): RequestHandler {
+export function createAppJsonMiddleware(providerVersion: string, runtimeVersion?: string, platform: boolean = false): RequestHandler {
     return async (req: Request, res: Response, next: NextFunction) => {
         const configPath = req.params[0];            // app.json path, relative to 'res' dir
+        const isProvider = configPath.includes('provider.json');
```

Comment:
Provider manifest is typically `/provider/app.json`, which wouldn't be caught by this check. Should also consider how non-service applications will work here (seems it should be fine, though?).

Etiher `configPath.includes('provider')`, or reading the manifest and checking the UUID?..